system-test/header.js

```diff
@@ -1,35 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Can we have this one back? I think it's useful. Synthtool should not remove it, should it?  We can convert it to TypeScript manually.